pulp_container/constants.py

```diff
@@ -19,6 +19,8 @@
     FOREIGN_BLOB_OCI_TAR_GZIP="application/vnd.oci.image.layer.nondistributable.v1.tar+gzip",
     FOREIGN_BLOB_OCI_TAR_ZSTD="application/vnd.oci.image.layer.nondistributable.v1.tar+zstd",
     OCI_EMPTY_JSON="application/vnd.oci.empty.v1+json",
+    HELM="application/vnd.cncf.helm.config.v1+json",
+    COSIGN="application/vnd.dev.cosign.simplesigning.v1+json",
```

Comment:
cosign_blob to be more precise?

Reply:
what about attestations and sboms?
these are also cosgn produced artifacts? how about you split these into cosign_sig_blob, cosign_att_blob and cosign_sbom_blob?

Reply:
see more examples here https://quay.io/repository/sallyom/hello-go?tab=tags
for sboms one can have spdx type and cyclonedx but you should rather check what's the situation as of today, my info might alreayd be outdated.

Reply:
Thank you for pointing the missing types! I also found this doc which was helpful to define the other cosign "derived types":  https://github.com/sigstore/cosign/blob/main/specs/SBOM_SPEC.md